lib/pages/timetable/view/events/event_widget.dart

```diff
@@ -1,101 +1,102 @@
-//        child: Column(
-//          crossAxisAlignment: CrossAxisAlignment.start,
-//          mainAxisSize: MainAxisSize.min,
-//          mainAxisAlignment: MainAxisAlignment.end,
-//          children: <Widget>[
-//            Padding(
-//              padding: const EdgeInsets.fromLTRB(4, 2, 4, 0),
-//              child: AutoSizeText(
-//                event.title ?? event.mainEvent?.classHeader?.acronym ?? '',
-//                maxLines: 2,
-//                minFontSize: 4,
-//                maxFontSize: 12,
-//                style: Theme.of(context).textTheme.bodyText2.copyWith(
-//                      fontSize: 12,
-//                      fontWeight: FontWeight.w600,
-//                      color: color.highEmphasisOnColor,
-//                    ),
-//              ),
-//            ),
-//            if (event.mainEvent.type != null)
-//              Expanded(
-//                child: Padding(
-//                  padding: const EdgeInsets.fromLTRB(4, 2, 4, 2),
-//                  child: AutoSizeText(
-//                    event.mainEvent.type.toLocalizedString(),
-//                    wrapWords: false,
-//                    minFontSize: 10,
-//                    maxFontSize: 10,
-//                    maxLines: 1,
-//                    overflow: TextOverflow.ellipsis,
-//                    style: Theme.of(context).textTheme.bodyText2.copyWith(
-//                          fontSize: 10,
-//                          color: color.highEmphasisOnColor,
-//                        ),
-//                  ),
-//                ),
-//              ),
-//            Expanded(
-//              child: footer?.isNotEmpty ?? false
-//                  ? Align(
-//                      alignment: Alignment.bottomRight,
-//                      child: Padding(
-//                        padding: const EdgeInsets.fromLTRB(4, 2, 4, 2),
-//                        child: AutoSizeText(
-//                          footer,
-//                          maxLines: 1,
-//                          minFontSize: 10,
-//                          overflow: TextOverflow.ellipsis,
-//                          style: Theme.of(context).textTheme.bodyText2.copyWith(
-//                                fontSize: 12,
-//                                color: color.mediumEmphasisOnColor,
-//                              ),
-//                        ),
-//                      ),
-//                    )
-//                  : Container(),
-//            ),
-//          ],
-//        ),
-//      ),
-//    );
-//  }
-//}
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:black_hole_flutter/black_hole_flutter.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:timetable/timetable.dart';
+
+import '../../model/events/uni_event.dart';
+import 'event_view.dart';
+
+/// Widget to display all day events in the timetable, based on
+/// [BasicEventWidget] from the timetable API.
```

Comment:
See my previous comment

Reply:
Not too many significant changes, apart from the null-safe operators https://www.diffchecker.com/YnuD7cTv